app/templates/components/crate-row.hbs

```diff
@@ -10,6 +10,7 @@
         {{#each crate.annotated_badges as |badge|}}
             {{component badge.component_name badge=badge}}
         {{/each}}
+        {{build-info crate=crate}}
```

Comment:
I think we should think about renaming this to `badge-build-info`, to help distinguish it from the tier1 build info (which will want to be distinguished from a future *all* build info)